src/FileSystem-Core.package/Path.class/class/from.delimiter..st

```diff
@@ -5,8 +5,10 @@ from: aString delimiter: aDelimiterCharacter
 	aString isEmpty
 		ifTrue: [ ^ self root ].
 	
-	pathClass :=  ((self isAbsoluteUnixPath: aString) or: [self isAbsoluteWindowsPath: aString]) 
+	pathClass :=  ((self isAbsolutePath: aString delimiter: aDelimiterCharacter) or: 
+							[self isAbsoluteWindowsPath: aString]) 
 		ifTrue: [ AbsolutePath ]
 		ifFalse:[ RelativePath ].
 	
-	^ pathClass withAll: (pathClass canonicalizeElements: (aDelimiterCharacter split: aString))
\ No newline at end of file
+	^ pathClass withAll: ((aDelimiterCharacter split: aString) select: 
```

Comment:
And here's the modified method:

from: aString delimiter: aDelimiterCharacter 
	"Answer a path composed of several elements delimited by aCharacter"
	| pathClass |
	aString isEmpty
		ifTrue: [ ^ self root ].
	
	pathClass :=  ((self isAbsolutePath: aString delimiter: aDelimiterCharacter) or: 
							[self isAbsoluteWindowsPath: aString]) 
		ifTrue: [ AbsolutePath ]
		ifFalse:[ RelativePath ].
	
	^ pathClass withAll: (pathClass canonicalizeElements: (aDelimiterCharacter split: aString))